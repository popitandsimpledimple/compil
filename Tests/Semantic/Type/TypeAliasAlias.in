type int = Integer;
type int_int = int;
const a: int_int = 1;
var b: int_int;
begin end.