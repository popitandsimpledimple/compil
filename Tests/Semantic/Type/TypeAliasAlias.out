(3,20) incompatible types: got 'integer' expected 'int_int'